src/makeRobot.js

```diff
@@ -38,7 +38,70 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
```

Comment:
```suggestion
  return {
```
No need to create a variable just to return it in the next step.